Header header
uint32 week
float64 seconds
float64 latitude
float64 longitude
float64 altitude
float64 undulation
float64 std_lat
float64 std_lon
float64 std_alt
float64 ve
float64 vn
float64 vu
float64 std_ve
float64 std_vn
float64 std_vu
float64 pitch
float64 roll
float64 yaw
float64 std_pitch
float64 std_roll
float64 std_yaw
uint32 ns
uint32 gnss_st
uint32 nav_st
uint32 odo_st
string nav_status
